fineract-provider/src/main/resources/db/changelog/tenant/changelog-tenant.xml

```diff
@@ -39,4 +39,5 @@
     <include file="parts/0017_fix_stretchy_reports.xml" relativeToChangelogFile="true"/>
     <include file="parts/0018_pentaho_reports_to_table.xml" relativeToChangelogFile="true"/>
     <include file="parts/0019_refactor_loan_transaction.xml" relativeToChangelogFile="true"/>
+    <include file="parts/0020_MyInterviewMigration.xml" relativeToChangelogFile="true"/>
```

Comment:
Interview?? is this an interview task?

Reply:
Hello @adamsaghy this PR was an interview task